src/QsCompiler/Tests.Compiler/TestCases/LinkingTests/Generics.qs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// This namespace contains test cases for generic operations
+namespace Microsoft.Quantum.Testing.Generics {
+
+	operation Test1Main() : Unit {
+		body (...) {
```

Comment:
nit: you could just drop the explicit specialization and go with
```suggestion
```
and remove the corresponding closing brace below.